recipes/string-view-lite/all/conandata.yml

```diff
@@ -5,3 +5,9 @@ sources:
   "1.4.0":
     url: https://github.com/martinmoene/string-view-lite/archive/v1.4.0.tar.gz
     sha256: 650c135be0bdc5bcae6876864b8279f15b1b97069da944ec2c773cacdea27e27
+  "1.5.1":
+    url: https://github.com/martinmoene/string-view-lite/archive/v1.5.1.tar.gz
```

Comment:
Do we need 1.5.1, if 1.6.0 is already available?

Reply:
Hmm it's header only so no, it's header only and costs very little. I'm in different, I'll remove it if you like

Reply:
I am indifferent as well.
Imho it's weird to add a version of which we know the chance that it will ever be used is slim to none.